packages/installer.vm/installer.vm.nuspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>installer.vm</id>
+    <version>0.0.0.20230904</version>
+    <title>Mandiant VM Installer</title>
```

Comment:
I would remove the Mandiant here everywhere. Non-Mandiant can use all of this as well and I don't want to rename a company name in this repo again. 😉